src/processors/downgrade-unmodified-lines.js

```diff
@@ -0,0 +1,15 @@
+const fs = require( 'fs' );
+const downgradeUnmodifiedLines = require( '../lib/downgrade-unmodified-lines' );
+const differ = require( '../lib/differ' );
+const gitDiffCalculator = require( '../lib/git-diff-calculator' );
+
+const config = JSON.parse( fs.readFileSync( '.eslines.json', 'utf-8' ) );
+
+module.exports = function( report ) {
+	const remote = config.processors[ 'downgrade-unmodified-lines' ].remote || 'origin/master';
```

Comment:
That approach won't allow configuring processors when they are used as ESLint formatters.

One option that we could explore is passing configuration only through env variables. So, at least if you use them as ESLint formatters you'll have a way to pass them config - although you'll need to parse the config yourself and convert it to env vars, so I'm not a fan of this either.

At the moment, I don't see any gains from removing the config parsing from processors other than negligible performance, so I believe this falls into the category of [PrematureOptimization](http://wiki.c2.com/?PrematureOptimization). I'll be very happy to revisit this decision if we come up with a solution that also works when they are used as ESLint formatters.